uniparc-rest/src/main/java/org/uniprot/api/uniparc/controller/UniParcEntryLightController.java

```diff
@@ -98,6 +99,9 @@ protected UniParcEntryLightController(
                             @Content(
                                     mediaType = APPLICATION_JSON_VALUE,
                                     schema = @Schema(implementation = UniParcEntryLight.class)),
+                                @Content(
+                                        mediaType = APPLICATION_XML_VALUE,
+                                        schema = @Schema(implementation = UniParcEntryLight.class)),
                             @Content(mediaType = RDF_MEDIA_TYPE_VALUE),
```

Comment:
The schema type should be jaxb one.
i guess something like "org.uniprot.core.xml.jaxb.uniparc.Entry.class"